### DISCUSSION TITLE
Filter by error in sub span

### DISCUSSION THREAD

**Discussion by bettlebrox** [langfuse/langfuse#3291] [Ideas]
### Describe the feature or potential improvement

Its currently possible to filter a trace by error in the sub generation. However its not possible to filter for traces that have an error in a span.

For example to find all traces with an error like this. 

![Screenshot 2024-09-10 at 14 16 34](https://github.com/user-attachments/assets/e4650141-d14a-4768-a0fc-16e4fd2eeb8a)

I had to download the CSV and filter using pandas.

### Additional information

related discussion:
https://github.com/orgs/langfuse/discussions/3079
https://github.com/orgs/langfuse/discussions/2888

**Comment by clemra** [comment]
Excellent feedback. cc'ing @marliessophie here who is currently doing some work on our tables. We'll keep this in mind + thank you for sharing your workaround in exporting csv and using pandas.

**Comment by marcklingen** [comment]
This is fixed, thanks again for sharing.

**Comment by bettlebrox** [comment]
Works for me. Thanks Marc

**Reply by marcklingen** [reply]
awesome, thanks for confirming
